app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PasswordsController < ApplicationController
+  # allows only logged in users
+  before_action :require_user_logged_in!
+  def edit; end
+  def update
+    # update user password
```

Comment:
Use proper spacing between method definitions.
```suggestion
  before_action :require_user_logged_in!

  def edit; end

  def update
    # update user password
```